lodash.js

```diff
@@ -8244,6 +8244,35 @@
     }
 
     /**
+     * Checks if `value` is `null` or `undefined`.
+     *
+     * @static
+     * @memberOf _
+     * @category Lang
+     * @param {*} value The value to check.
+     * @returns {boolean} Returns `true` if `value` is `null` or `undefined`, else `false`.
+     * @example
+     *
+     * _.isNil(null);
+     * // => true
+     *
+     * _.isNil();
+     * // => true
+     *
+     * _.isNil(undefined);
+     * // => true
+     *
+     * _.isNil(NaN);
+     * // => false
+     *
+     * _.isNil(void 0);
+     * // => false
+     */
+    function isNil(value) {
+      return undefined === value || null === value;
```

Comment:
The yoda conditions aren't really our style. Could you make this `return value == null;`.